display/ssd1306.h

```diff
@@ -225,15 +229,37 @@ class SSD1306 : public I2CDevice, Looper, StateMachine, SaberBase, private Audio
 	}
 	if (eof_) {
 	  // STDOUT << "EOF " << frame_count_ << "\n";
-	  screen_ = SCREEN_PLI;
-	  if (frame_count_ == 1) return 5000;
-	  return FillFrameBuffer();
+    if (!SaberBase::IsOn()) {
```

Comment:
definitely something wrong with indentation still.
If there are tabs in here, please remove them.
If that is not the problem, I'm going to have to fix the indentation after I merge.
(Assuming of course that it's not the diff tool that is just showing it weird...)

Reply:
Any tabs were present when I pulled the file from github, my editor is configured to use spaces.